flixel/effects/postprocess/PostProcess.hx

```diff
@@ -61,21 +62,21 @@ class PostProcess extends OpenGLView
 
 		buffer = GL.createBuffer();
 		GL.bindBuffer(GL.ARRAY_BUFFER, buffer);
-		GL.bufferData(GL.ARRAY_BUFFER, new Float32Array(cast vertices), GL.STATIC_DRAW);
+		GL.bufferData(GL.ARRAY_BUFFER, new Float32Array(#if !openfl_next cast #end vertices), GL.STATIC_DRAW);
```

Comment:
The new typed arrays implementation chooses how to construct the array from the type of the argument, having it as an unsafe cast breaks it.